Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build test run clean deps help install-local package binaries generate
+.PHONY: build build-frontend test run clean clean-frontend deps help install-local package binaries generate
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)